SupervisedTestsRunner.js

```diff
@@ -46,7 +46,7 @@ class SupervisedTestsRunner {
   startAllTests() {
     return this.calculateTestFiles().then(async ({ files, filterEnabled }) => {
       const testsForRun = this.config.filterTestsFiles
-        ? await this.config.filterTestsFiles(files, filterEnabled)
+        ? await this.config.filterTestsFiles(files, filterEnabled, this.config)
```

Comment:
Hmmm.. wait. Why do exactly we need such a change? `this.config` is the actual config we pass to the frontend-tests-runner, is not it?

I mean, if we have code like this:

```js
const config = {
  // ...
  someOption: 'some value',
  filterTestsFiles: (files, isFilteredByOnly) => {
    // config.someOption is available here
  }
};
```

as you see, `config` is available there.

If we do not have a variable for the config, e.g. like this:

```js
module.exports = {
  // ...
  someOption: 'some value',
  filterTestsFiles: function (files, isFilteredByOnly) {
    // this.someOption is available here
  }
};
```

We still can access an option from the config.

So, my question is, why do we need to pass the config here? 🤔

Reply:
Alright, we've decided that it would be just handier to pass the config this way.